src/Uno.UI/ts/Windows/Storage/AssetManager.ts

```diff
@@ -0,0 +1,24 @@
+﻿
+// eslint-disable-next-line @typescript-eslint/no-namespace
+namespace Windows.Storage {
+
+	export class AssetManager {
+		public static async DownloadAssetsManifest(path: string): Promise<string> {
+			var response = await fetch(path);
```

Comment:
Codacy found an issue: [Identifier 'response' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6212827)